metashare/api/management/commands/resync_all_gh_commit_data.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+
+from ...jobs import refresh_commits
+from ...models import Task
+
+
+class Command(BaseCommand):
+    help = "Remove and resync all stored commits from GitHub."
+
+    def handle(self, *args, **options):
+        for task in Task.objects.filter(
+            branch_name__isnull=False, origin_sha__isnull=False
+        ):
+            refresh_commits(
+                repository=task.project.repository, branch_name=task.branch_name
+            )
```

Comment:
This should probably avoid doing the same work twice if there are multiple tasks with the same repo/branch combo

Reply:
We expect that to not ever be the case under normal circumstances.

Reply:
Right, that should not even be possible. Also, this command can only be run manually currently.